Src/IronPythonTest/Cases/AllCPythonCasesManifest.ini

```diff
@@ -2,7 +2,7 @@
 Ignore=false
 Arguments="$(TEST_FILE)"
 WorkingDirectory=$(ROOT)/Src/StdLib/Lib
-Redirect=false
+Redirect=true
```

Comment:
You wanted to redirect for all tests?

Reply:
Disregard, I missed the only running test_subprocess in this for debugging.

Reply:
As long as it's not causing issues with a specific test then I think we should.